packages/astro/src/core/render/result.ts

```diff
@@ -182,9 +182,9 @@ export function createResult(args: CreateResultArgs): SSRResult {
 				request,
 				url,
 				redirect: args.ssr
-					? (path: string) => {
+					? (path: string, status: 301 | 302 | 307 = 302) => {
```

Comment:
Thoughts on making this an options object so it is more extensible in the future? 
```suggestion
					? (path: string, { status = 302 }: { status?: 301 | 302 | 307 } = {}) => {
```